config/locales/server.en.yml

```diff
@@ -9,6 +9,54 @@ en:
      channel_exists_for_category: "A channel already exists for this category and name"
      channel_exists_for_topic: "A channel already exists for this topic"
 
+    reviewables:
+      actions:
+        agree:
+          title: "Agree..."
+        agree_and_keep_message:
+          title: "Keep Message"
+          description: "Agree with flag and keep the message unchanged."
+        agree_and_keep_deleted:
+          title: "Keep Message Deleted"
+          description: "Agree with flag and leave the message deleted."
+        agree_and_suspend:
+          title: "Suspend User"
+          description: "Agree with flag and suspend the user."
+        agree_and_silence:
+          title: "Silence User"
+          description: "Agree with flag and silence the user."
+        agree_and_restore:
+          title: "Restore Message"
+          description: "Restore the message so that users can see it."
+        agree_and_delete:
+          title: "Delete Message"
+          description: "Delete the message so that users cannot see it."
+        delete_single:
+          title: "Delete"
+        delete:
+          title: "Delete..."
+        delete_and_ignore:
+          title: "Delete Message and Ignore"
+          description: "Delete message; if the first message, delete the topic as well"
+        delete_and_ignore_replies:
```

Comment:
You're not using all these keys are you? This is suspicious to me:

> "Delete message and all of its replies; if the first message, delete the topic as well"

Reply:
good catch